platforms/js/build_js.py

```diff
@@ -194,6 +196,7 @@ def build_doc(self):
     parser.add_argument('--skip_config', action="store_true", help="Skip cmake config")
     parser.add_argument('--config_only', action="store_true", help="Only do cmake config")
     parser.add_argument('--enable_exception', action="store_true", help="Enable exception handling")
+    parser.add_argument('--custom_flags', help="Adds the ability to modify the list of modules") 
```

Comment:
1. `custom_flags` => `cmake_option`
2. `.split(" ")` approach is not reliable. What is about list support? [`action='append'`](https://docs.python.org/2/library/argparse.html#action)